### PR TITLE
Update and rename CCC.xml to CCC.de.xml

### PR DESCRIPTION
**Pull request by koops76**

### CODE REVIEW

File: src/chrome/content/rules/CCC.de.xml [hunk 1]
```diff
@@ -1,57 +1,44 @@
...
-
-	¹ Shows wuppa
-
-	Problematic hosts in *ccc.de:
-
-		- wiki.chaosradio²
-		- ds ¹
-		- dsrad ¹
-		- wuppa ¹
-		- irc.darmstadt
```
Comment (J0WI) [inline src/chrome/content/rules/CCC.de.xml]: You missed this comment.

Comment (koops76) [reply src/chrome/content/rules/CCC.de.xml]: Which issue does it have?

Comment (MilanKral) [reply src/chrome/content/rules/CCC.de.xml]: Please include also https://ds.ccc.de/
http://noc.events.ccc.de/ includes the same content as  https://ds.ccc.de/

Comment (koops76) [reply src/chrome/content/rules/CCC.de.xml]: @MilanKral Done.

Comment (J0WI) [reply src/chrome/content/rules/CCC.de.xml]: @koops76 just try to access to get the issue: https://irc.darmstadt.ccc.de/

Comment (koops76) [reply src/chrome/content/rules/CCC.de.xml]: @J0WI Done.

Comment (J0WI) [reply src/chrome/content/rules/CCC.de.xml]: Please have a look at Travis.

File: src/chrome/content/rules/CCC.de.xml [hunk 2]
```diff
@@ -1,58 +1,47 @@
...
-	² no valid cert
-	³ incomplete chain
-
-	403:
-		-koeln.media
-
-	Timeout:
-		- noc.events
+	Dropped:
+		- noc.events (fixed by this ruleset)
```
Comment (J0WI) [inline src/chrome/content/rules/CCC.de.xml]: This seems not to be `fixed by this ruleset`

Comment (koops76) [reply src/chrome/content/rules/CCC.de.xml]: @J0WI Done.

**Comment by J0WI** [comment]
Please have a look at Travis again.

**Comment by koops76** [comment]
@J0WI Done.

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**